### PR TITLE
Fix hero centering and form width

### PR DESCRIPTION
**Pull request by Magnus-G**
image

image

### CODE REVIEW

File: tmp/app/assets/stylesheets/refills/_vertical-tabs.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
+.vertical-tabs-container {
+  $large-screen: em(640) !default;
```
Comment (brendastorer) [inline tmp/app/assets/stylesheets/refills/_vertical-tabs.scss]: Now may not be the time to adjust, but I believe this px -> em mixin calculator is being deprecated in Bourbon.

**Comment by brendastorer** [comment]
:money_with_wings: :money_with_wings: :money_with_wings:

File: tmp/app/assets/stylesheets/refills/_vertical-tabs.scss (hunk 1 above)
Comment (Magnus-G) [inline tmp/app/assets/stylesheets/refills/_vertical-tabs.scss]: Ah! Good to know. So we just write an `em` value?

Comment (brendastorer) [inline tmp/app/assets/stylesheets/refills/_vertical-tabs.scss]: Yep, or the px value, if you dare.

**Pull request closed without merging**